packages/grafana-ui/src/types/panel.ts

```diff
@@ -27,6 +27,7 @@ export interface PanelProps<T = any> {
   // TODO: annotation?: PanelData;
 
   timeRange: TimeRange;
+  timeZone: TimeZone;
```

Comment:
I added TimeZone here as it is needed in the Graph to display correct values on axis. Not sure this is the correct way.

Reply:
think your right that this is needed